src/content/en/updates/posts/2016/03/device-mode-v2.markdown

```diff
@@ -0,0 +1,112 @@
+
+While we offered ways to emulate mobile in the past, the development default has 
+been Desktop: Emulation always had to be switched on. Now that consumption of 
+mobile sites has overtaken Desktop in many places, we're switching our position 
+in DevTools as well. **Mobile is becoming the starting point in Chrome 
+DevTools**.
+
+## What's new?
+
+![New Device Mode](/web/updates/images/2016/03/device-mode-v2/device-mode-initial-view.png)
+
+First and foremost, the UI is streamlined and uses a lot less space. We expect 
+the new Device Mode to become the main development mode for most, so a clean and 
+simple design that extends the main DevTools navigation bar was a requirement.
+
+![New Device Mode](/web/updates/images/2016/03/device-mode-v2/media-query-inspector-ruler.png)
+_The new quick-jump device ruler over the media queries._
+
+In addition, we've centered the viewport and added a new quick-jump device ruler 
+at top, a great help when designing responsively, that gives you an idea of the 
+most common device sizes. 
+
+And finally, a lot of options have been bundled or hidden behind a toggle 
+whenever possible. These new composite options make it a lot easier to switch 
+between modes. To toggle certain controls or customize your experience of the 
+toolbar, hit up the little three dot menu icon.
+
+## Responsive by default
+
+<img src="/web/updates/images/2016/03/device-mode-v2/select-device.png" alt="Device Mode dropdown" style="float: right;max-width: 33%;margin: 0 0 10px 10px;">
+
+The main DevTools toolbar now expands to the left side of the browser window and 
+includes the most important tools to emulate a variety of mobile and Desktop 
+devices. You can choose between two development modes:
+
+* **Responsive**
+* **Specific Device**
+
+In both modes, the viewport sits in its own resizable window within Chrome.This 
+has the significant advantage that you can maximize your browser window and the 
+DevTools the way you like them and not have them jump around when you test 
+multiple sizes of your page and go back and forth.
+
+**Responsive** is the mode you'll want to be in during active iteration to make 
+sure your site works on all sorts of devices, not just a few specific ones. In 
+this mode, the handles next to the viewport are freely resizable.
+
+**Specific Device** refers to when you choose a specific device and lock the 
+viewport to its size. This becomes useful when you want to get in final fixes 
+and touches for a few popular devices near the launch. Which is why we're not 
+just showing a huge list of all sorts of devices in the dropdown, but the 
+currently most popular ones. If you select one, we do our best to make it behave 
+as closely as the real deal: Touch events, user agent, viewport and device 
+chrome and UI (if available) are all emulated.
+
+## Integrated Remote Debugging
+
+Emulation, even the best one available, can only get you so far. There are 
```

Comment:
Emulations, even the best ones available, can only get you so far. There are simply things that emulations can't do today, like: